content/pages/Donate.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Donate
+blocks:
+  - logo:
+      src: /inreach-logos/InReach_Logo_Color_RGB.svg
+      alt: InReach Logo
+    navBGColor: Green
+    navLinks:
+      - linkName: About
+        linkUrls:
+          - sectionUrl: Mission
+            pathURL: /
+          - sectionUrl: Our Team
+            pathURL: /example-gallery
+          - sectionUrl: Our Vetting Process
+            pathURL: /
+      - linkName: News
+        linkUrls:
+          - sectionUrl: In The News
+            pathURL: /
+          - sectionUrl: Awards
+            pathURL: /
+          - sectionUrl: Our Blog
+            pathURL: /
+      - linkName: Get Help
+      - linkName: Take Action
+      - linkName: Contact
+    _template: header
+  - heroBackgroundImage:
+      src: /Donate/Donatepage.jpg
+      alt: Donate
+      colorFilter: '#6C6A6A'
+    heroHeaderText:
+      text: |
+        # <ColoredText text="DONATE" color="White" />
+      position: Left
+    _template: hero
+  - richTextLeft: "<YoutubeEmbed url=\"https://www.youtube.com/watch?reload=9&v=l_sm8Q1r3d8\" />\n\nYour donation matters! Here’s the impact you can make:\n\n\U0001F49A $25 = 21 more people supported\n\U0001F49A $50 = 42 more people supported\n\U0001F49A $100 = 84 more people supported\n\U0001F49A $250 = 210 more people supported\n\U0001F49A $500 = 420 more people supported\n\U0001F49A $1,000 = 840 more people supported\n\nEvery contribution ensures #SafeHelpIsInReach for our diverse LGBTQ+ community. \U0001F308\U0001F3F3️‍⚧️\n"
```

Comment:
I believe the youtube video id is incorrect. doesn't play anything

<img width="1482" alt="Screenshot 2024-12-17 at 11 33 56 PM" src="https://github.com/user-attachments/assets/622fd59c-db3f-4ae9-b08f-c8057312be98" />